src/features/world/ui/npcs/Hopper.tsx

```diff
@@ -0,0 +1,79 @@
+import { SUNNYSIDE } from "assets/sunnyside";
+import { Label } from "components/ui/Label";
+import { CloseButtonPanel } from "features/game/components/CloseablePanel";
+import React, { useContext } from "react";
+import { InlineDialogue } from "../TypingMessage";
+import { NPC_WEARABLES } from "lib/npcs";
+import { SpecialEventModalContent } from "../SpecialEventModalContent";
+import { Context } from "features/game/GameProvider";
+import { useActor } from "@xstate/react";
+import { SpecialEvent } from "features/game/types/specialEvents";
+import { useAppTranslation } from "lib/i18n/useAppTranslations";
+
+interface Props {
+  onClose: () => void;
+}
+
+// Store per session
+let RABIBTS_FOUND = 0;
+
+const RABBITS = 6;
+
+export function collectRabbit() {
+  RABIBTS_FOUND += 1;
+}
+
+export const Hopper: React.FC<Props> = ({ onClose }) => {
+  const { gameService } = useContext(Context);
+  const [gameState] = useActor(gameService);
+
+  const { t } = useAppTranslation();
+
+  if (RABIBTS_FOUND < RABBITS) {
+    return (
+      <CloseButtonPanel bumpkinParts={NPC_WEARABLES.hopper}>
```

Comment:
This modal doesn't allow you to close it once Hopper asks you to help find the bunnies. You can click out of the modal but I found myself clicking inside the modal a couple of times wondering why it wasn't closing. 🤷🏼